Gemfile

```diff
@@ -35,6 +35,7 @@ group :development do
   gem "spring-commands-rspec"
   gem "web-console"
   gem "warden"
+  gem 'bcrypt'
```

Comment:
Yeah we do good find ! unfortunately I forgot my macbook charger at Defacto so I
can't fix this stuff tonight :S
On Mon, Oct 26, 2015 at 9:02 PM, DefactoSoftware/proof
<reply@reply.github.com>
wrote:
In Gemfile [https://github.com/DefactoSoftware/proof/pull/12#discussion_r43044645] :

> @@ -35,6 +35,7 @@ group :development do
>    gem "spring-commands-rspec"
>    gem "web-console"
>    gem "warden"
> +  gem 'bcrypt'


we also need warden and bcrypt in production right?

—
Reply to this email directly or view it on GitHub
[https://github.com/DefactoSoftware/proof/pull/12/files#r43044645] .[https://github.com/notifications/beacon/ABKd9PT9DMlJwf5SYfLMGu7JXzve7wlRks5o_n5NgaJpZM4GVkDq.gif]